pkg/cache/informer_cache.go

```diff
@@ -71,6 +71,9 @@ func (ip *informerCache) Get(ctx context.Context, key client.ObjectKey, out clie
 // List implements Reader
 func (ip *informerCache) List(ctx context.Context, out client.ObjectList, opts ...client.ListOption) error {
 
+	listOpts := client.ListOptions{}
+	listOpts.ApplyOptions(opts)
```

Comment:
I don't understand how this changes anything, this variable is never accessed or am I completely missing something?

Reply:
No, you're right, this was a very silly oversight on my part, I don't remember why I didn't implement the remainder of it, I apologize. I will fix it.